server/lib/chat/chat-service.ts

```diff
@@ -127,8 +132,8 @@ export default class ChatService {
       this.publisher.publish(getChannelPath(originalChannelName), {
         action: 'join2',
         user: {
-          id: result.userId,
-          name: result.userName,
+          id: userInfo.id,
```

Comment:
nit: just return the result from `findUserById` directly, so we don't have to update this every time that struct changes